### PR TITLE
docs: describe CLI

### PR DESCRIPTION
**Pull request by KuznetsovRoman**

### CODE REVIEW

File: docs/commands/browser/switchToRepl.mdx
Comment (KuznetsovRoman) [inline docs/commands/browser/switchToRepl.mdx:1]: Создал этот пустой файл, чтобы docusaurus собрался

File: i18n/en/docusaurus-plugin-content-docs/current/commands/browser/switchToRepl.mdx
Comment (KuznetsovRoman) [inline i18n/en/docusaurus-plugin-content-docs/current/commands/browser/switchToRepl.mdx:1]: I created this empty file so docusaurus could build

File: docs/command-line/index.mdx [hunk 1]
```diff
@@ -0,0 +1,251 @@
+import Admonition from "@theme/Admonition";
+
+# Интерфейс командной строки {#cli}
```
Comment (shadowusr) [inline docs/command-line/index.mdx:3]: I suggest CLI or Testplane CLI instead, subjectively my eyes are trained to look for "CLI" on the page

Same goes for the en version

**Review by shadowusr — APPROVED** [review]
:fire:

**Pull request merged**